### PR TITLE
Hw jump and modify

### PR DESCRIPTION
**Pull request by jiaweiwsz**
For example:

table 0x5b499a0, type FDB, ufid 0x5bf96d0   key: in_port: sf0     action: METER & MODIFY_HDR(hdr(set_reg_c3=0x8000)) & JUMP to Next Table 0x5bf1080
table 0x5bdca00, type FDB, ufid 0x5bfaec0   key: in_port: sf0    action: MODIFY_HDR(hdr(set_reg_c2=0xf0000)) & counter(hits(704), bytes(119944)) & JUMP to Next Table 0x5bf9730
table 0x5bdcc60, type FDB, ufid 0x5bf6f00   key: metadata_reg_c_3: 0x00008000 metadata_reg_c_3: 0x00008000    action: counter(hits(0), bytes(0)) & DROP
table 0x5bdcc60, type FDB, ufid 0x5bee670   key: metadata_reg_c_3: 0x00000100 metadata_reg_c_3: 0x00000100    action: counter(hits(0), bytes(0)) & DROP
table 0x5bdcc60, type FDB, ufid 0x5bed3b0   key:      action: DROP
table 0x5bdf720, type FDB, ufid 0x5be0bc0   key: metadata_reg_c_3: 0x00000002 metadata_reg_c_3: 0x00000002    action: JUMP to Next Table 0x5bdca00
table 0x5bdf720, type FDB, ufid 0x5be0c90   key:      action: JUMP to Next Table 0x5bdcc60
table 0x5bf1080, type FDB, ufid 0x5bf82d0   key: metadata_reg_c_3: 0x00000002, in_port: sf0     action: METER & MODIFY_HDR(hdr(set_reg_c3=0x100)) & JUMP to Next Table 0x5bdf720
table 0x5bf1080, type FDB, ufid 0x5bf3980   key:      action: JUMP to Next Table 0x5bdcc60
table 0x5bf1080, type FDB, ufid 0x5bf2610   key: metadata_reg_c_3: 0x00000002, in_port: pf     action: MODIFY_HDR(hdr(set_reg_c3=0x100)) & METER & JUMP to Next Table 0x5bdf720
table 0x5bf9730, type FDB, ufid 0x5bfc280   key: metadata_reg_c_2: 0x000f0000, in_port: sf0    action: output(pf)

### CODE REVIEW

File: src/dr_action.py [hunk 1]
```diff
@@ -130,21 +130,20 @@ def add_dump_ctx(self, dump_ctx):
         self.dump_ctx = dump_ctx
 
     def dump_str(self):
+        if simple_output:
+            if ( (_srd(self.data, "id")) in self.dump_ctx.modify_hdr.keys()):
+                out_str = self.dump_ctx.modify_hdr[(_srd(self.data, "id"))].lstrip(',')
+                return "MODIFY_HDR(hdr(%s))" % (out_str)
+
```
Comment (haifei0926) [inline src/dr_action.py:137]: Line 137: add:
       else return "MODIFY_HDR"

Comment (jiaweiwsz) [reply src/dr_action.py:137]: OK

Comment (haifei0926) [reply src/dr_action.py:139]: i think  "else" is for if ( (_srd(self.data, "id")) in self.dump_ctx.modify_hdr.keys()):  . IF we do not find the id in key, then simply print "MODIFY_HDR".

Comment (jiaweiwsz) [reply src/dr_action.py:139]: right, wrong indent. fixed.

**Pull request merged**